DSharpPlus.Commands/Processors/SlashCommands/SlashCommandProcessor.cs

```diff
@@ -284,7 +284,9 @@ public async Task<DiscordApplicationCommand> ToApplicationCommandAsync(Command c
             description_localizations: descriptionLocalizations,
             allowDMUsage: command.Attributes.Any(x => x is AllowDMUsageAttribute),
             defaultMemberPermissions: command.Attributes.OfType<RequirePermissionsAttribute>().FirstOrDefault()?.UserPermissions ?? Permissions.None,
-            nsfw: command.Attributes.Any(x => x is RequireNsfwAttribute)
+            nsfw: command.Attributes.Any(x => x is RequireNsfwAttribute),
+            contexts: command.Attributes.OfType<SlashAllowedContextsAttribute>().FirstOrDefault()?.AllowedContexts ?? [],
+            integrationTypes: command.Attributes.OfType<SlashInstallTypeAttribute>().FirstOrDefault()?.InstallTypes ?? []
```

Comment:
Both contexts and integrationTypes should default to null or contain defaults ([Guild, BotDM,PrivateChannel] and  [GuildInstall])

Reply:
the defaults i mentioned are the defaults discord uses when providing null but i think it would be valid to explicitly set the default because we all know that discord likes to change things on production and we would provide more consistency to our users